configurator/src/main/java/com/devonfw/ide/configurator/merge/DirectoryMerger.java

```diff
@@ -94,12 +97,18 @@ public void inverseMerge(File workspaceFile, VariableResolver resolver, boolean
         Log.LOGGER.warning("Workspace is missing directory: " + workspaceFile);
         return;
       }
+      Log.LOGGER.fine("Traversing directory " + updateFile);
       for (File child : updateFile.listFiles()) {
-        inverseMerge(workspaceFile, resolver, addNewProperties, new File(updateFile, child.getName()));
```

Comment:
98 was a severe bug. For recursion workspaceFile was never changed. However, directory traversal also has to be applied here so the child file or folder has to be passed instead of `workspaceFile`. Hence this code never worked.